src/samples/audio/smart_amp_test_ipc4.c

```diff
@@ -44,7 +44,7 @@ static int smart_amp_init(struct processing_module *mod)
 	struct smart_amp_data *sad;
 	struct comp_dev *dev = mod->dev;
 	struct module_data *mod_data = &mod->priv;
-	int bs;
+	int in_size, out_size;
```

Comment:
These are compile-time constants, not variables.
```suggestion
	const int in_size = sizeof(sad->ipc4_cfg.input_pins);
```